android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/AdjustLongpressDelayActivity.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (C) SIL International. All rights reserved.
+ */
```

Comment:
```suggestion
/*
 * Keyman is copyright (C) SIL International. MIT License.
 * 
 * (Optional description of this file)
 */
```

https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide#code-format